src/TesterInternal/TesterInternal.csproj

```diff
@@ -283,8 +296,14 @@
     <Content Include="ClientConfig_NewAzure.xml">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="ClientConfig_StreamProviders.xml">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
     <Content Include="Config_ActivationCollectorTests.xml">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Config_AzureTableStorage.xml">
```

Comment:
Is this file linked twice? You can see that it is also in line 350 of the previous file (or 372 in the new one): https://github.com/sergeybykov/orleans/commit/3c7ee8d0a363e65775a7c41ffd9f7b6b6b449890#diff-5560660fd5e52220b16a95f97107cdc2L350